README.rst

```diff
@@ -26,9 +26,9 @@ Dependencies
 ============
 PyOTA is compatible with Python 3.7, 3.6, 3.5 and 2.7
 
-============
-Installation
-============
+=============
+Install PyOTA
```

Comment:
This change is needed to make all section titles unique due to `autosectionlabel` sphinx extension. There is a section with the same title in `getting_started.rst`.

Reply:
💭 Hmm... wondering if we should configure Travis build so that it also tries to build the docs, so that we can catch these automatically.